.gitignore

```diff
@@ -41,3 +41,5 @@ dist
 
 # Node dependencies
 node_modules
+/.vscode
```

Comment:
Checking in on these being added in @zeus2611 :) We do have `settings.json` and `extensions.json` version controlled, so should we maybe take these lines out?

Reply:
I initially thought this folder was automatically modified and added to version control. However, since these files are already version-controlled, I’ll update the `.gitignore` file to exclude them. Thanks for pointing that out!